_includes/banner.html

```diff
@@ -3,56 +3,38 @@
     <div class="usa-banner__header">
       <div class="usa-banner__inner container">
         <div class="grid-col-auto">
-          <img
-            class="usa-banner__header-flag"
-            src="{{ site.baseurl }}/assets/img/us-flag.png"
-            alt="U.S. flag"
-          />
+          <img class="usa-banner__header-flag" src="{{ site.baseurl }}/assets/img/us-flag.png" alt="U.S. flag">
```

Comment:
I have a personal style preference for the self closing tag (`<img />`), @aduth @cannandev do we have a lint/policy around those?

```suggestion
          <img class="usa-banner__header-flag" src="{{ site.baseurl }}/assets/img/us-flag.png" alt="U.S. flag" />
```

Reply:
> @aduth @cannandev do we have a lint/policy around those?

No lint or policy that I'm aware of, and no strong preference from me. If it were me writing it on my own personal time, I'd have implemented it as originally proposed.

If anything, there could be a small argument that since the [USWDS sample code](https://designsystem.digital.gov/components/banner/) aligns to what was originally proposed, it makes it easier to copy directly from documentation for common components.

Reply:
Seems like a good enough reason to leave it to match the USWDS